ingest/scripts/override_group_segments.py

```diff
@@ -180,7 +180,7 @@ def write_grouped_metadata(
     groups: Groups,
 ) -> tuple[dict, set]:
     # Map from original accession to the new concatenated accession
-    fasta_id_map: dict[Accession, SubmissionId] = {}
+    fasta_id_map: dict[Accession, id] = {}
```

Comment:
```suggestion
    fasta_id_map: dict[Accession, Id] = {}
```

Type should be capitalized?

Reply:
Thanks - yeah! Feels like CI should have caught this (not necessarily the capitalisation thing, but the fact that `id` didn't exist as I had fixed the capitalisation above).

Reply:
we're not running type checking in CI yet - though some copilot AI should have! 😁